sonaranalyzer-dotnet/src/targets/CommonProperties.targets

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you put this content in `Directory.Build.props`? You can remove the explicit reference from the csproj files.

Reply:
Nice! This simplified the PR quite a bit.